mappings/net/minecraft/world/gen/chunk/StructureConfig.mapping

```diff
@@ -1,8 +1,29 @@
 CLASS net/minecraft/class_5314 net/minecraft/world/gen/chunk/StructureConfig
+	COMMENT Contains parameters for placement of a single type of {@link net.minecraft.world.gen.feature.StructureFeature} during chunk
+	COMMENT generation.
 	FIELD field_24917 CODEC Lcom/mojang/serialization/Codec;
 	FIELD field_24918 spacing I
+		COMMENT Defines how many chunks make up one cell in the grid used to place structures.
```

Comment:
Might be a bit confusing here- maybe rather:
```suggestion
		COMMENT Defines the width in chunks of one cell in the grid used to place structures.
```

Reply:
It's also the height though :-P

Reply:
True :P 
`size` could potentially work here also

Reply:
Changed it to "Defines the width and height of a cell in the structure placement grid in chunks."